From leemgs1@gmail.com  Fri Sep 11 15:05:47 2009
From: GeunSik Lim <leemgs1@gmail.com>
Date: Mon, 07 Sep 2009 21:39:25 +0900
Subject: debugfs: Modify default debugfs directory for debugging pktcdvd.
To: Greg KH <greg@kroah.com>
Cc: <balagi@justmail.de>
Message-ID: <1252327165.2153.102.camel@centos51>


As we all know, We need change default directory for consistency of
debugfs by Greg K-H

Signed-off-by: GeunSik Lim <geunsik.lim@samsung.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/block/pktcdvd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/block/pktcdvd.c
+++ b/drivers/block/pktcdvd.c
@@ -92,7 +92,7 @@ static struct mutex ctl_mutex;	/* Serial
 static mempool_t *psd_pool;
 
 static struct class	*class_pktcdvd = NULL;    /* /sys/class/pktcdvd */
-static struct dentry	*pkt_debugfs_root = NULL; /* /debug/pktcdvd */
+static struct dentry	*pkt_debugfs_root = NULL; /* /sys/kernel/debug/pktcdvd */
 
 /* forward declaration */
 static int pkt_setup_dev(dev_t dev, dev_t* pkt_dev);
